x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/hooks/use_endpoint_action_items.tsx

```diff
@@ -58,9 +60,9 @@ export const useEndpointActionItems = (
         selected_endpoint: endpointId,
       });
 
-      return [
-        isIsolated
-          ? {
+      const isolationActions = isIsolated // Un-isolate is always available to users regardless of license level
```

Comment:
Not needs to be changed, but when having more than one condition I like to use something like if/else instead of the conditional (ternary) operator.
In this case, if/else would be more readable:

```
let isolationActions = [];
if (isIsolated) {
  isolationActions = ....;
} else if (isPlatinumPlus) {
  isolationActions = ....;
} else if (whateverInTheFuture) {
  ...
}
```

Reply:
Yeah, much more legible 👍 
If the build fails for anything, I will make this change or add it to my next PR

Reply:
Done 😄